src/qtiItem/core/Loader.js

```diff
@@ -400,9 +400,13 @@ var Loader = Class.extend({
         const attributes = _.defaults(data.attributes || {}, element.attributes || {});
         element.setAttributes(attributes);
 
-        if (element.body && data.body) {
+        let body = data.body;
+        if (!body && data.text && data.qtiClass === 'inlineChoice') {
+            body = {body: data.text, elements: []};
```

Comment:
**question:** Should not `elements` be an object?
**suggestion:**
```suggestion
            body = { body: data.text, elements: {} };
```

Reply:
@jsconan as this was created by @wazelin and he is BE if you think this is better then i will sign off and commit the suggestion

Reply:
Works either way I suppose.

Reply:
But JFYI, it was an array initially.

<img width="804" alt="image" src="https://user-images.githubusercontent.com/2943256/221201206-cd8a6de1-fee1-4aec-ad6a-79f77fb8ac04.png">